### DISCUSSION TITLE
How about a new footer

### DISCUSSION THREAD

**Discussion by SuyashRajput** [danielschuster-muc/potter-db#776] [Ideas]
The old footer looks kind of basic
This is one of the footer I made in one of my website
![image](https://github.com/danielschuster-muc/potter-db/assets/110171204/c4590961-1eaf-46a3-9246-c4a673cc65f2)
I can make a good footer for you too :)

**Comment by danielschuster-muc** [comment]
Do you mean the footer of the [old website](https://potterdb.com/)? There won't be any major changes to the old website as it will get replace soon by the new website. 

If you refer to the [new website](https://beta.potterdb.com/) I am open for suggestions about the footer. What exactly would you have in mind?

**Comment by SuyashRajput** [comment]
@danielschuster-muc  I am talking about a footer with the main logo ( and some tag line if you want )
With social links and all the current links
I will make sure to make it responsive too

If you have any other suggestion in mind I am open to Ideas.

**Reply by danielschuster-muc** [reply]
I prefer a more simple style for the footer, but I am still interested in seeing your ideas. Maybe you could make some design ideas and share them here? 
Also regarding socials, I don't think this is really needed, as we basically only have the github link which would fall under this condition.
